README.md

```diff
@@ -1,4 +1,174 @@
-### Running notebooks
+## API
```

Comment:
I think there should be a small section at the beginning describing in a few sentences what this repository is about. Something like:
___
# openEO process graph to evalscript converter
This repository contains a library that converts an openEO process graph into a SH evalscript (as far as possible).

Enabling the openEO backend to interact with SH via evalscripts has several advantages compared to executing the whole process graph directly within the openEO backend. 
1) Part of the processing is done directly at the backend where the data is stored (SH) so less data has to be transfer between the SH backend and the openEO backend reducing the time and costs of process executions.
2) ???
_____

Is there any other advantage to add? The question is also if this approach is faster. On the plus side our SH backend should hopefully be really fast in executing evalscripts on the minus side the evalscript hast to be generated from a process graph first which takes some time.

Main advantage for sure is, that less data has to be transferred between backends.

Reply:
It's a standalone library, I'm not sure it makes sense talking about the backends here.

Reply:
But a standalone library that uses as input openEO process graphs and produces as output SH evalscripts so 2 very backend specific things. We can also keep it more generic but I think it would be useful to have a small introduction/give some context of the beginning of the README, what this library actually does.

Reply:
> some context of the beginning of the README, what this library actually does

Well, this it does have - it converts openEO process graphs to SH evalscripts. I really wouldn't be putting in "advantages", if we really have to we can put in a sentence about the motivation behind it, e.g. "the motivation behind this library is to reduce the data transfer between the SH backend and the openEO backend", but I'd keep it at that.

Reply:
I am fine with adding this short sentence about the motivation to give some more context. If you like you can still add "... and the openEO backend and to move part of the processing directly to backend where the data is stored."

FFTR and merge after you added this.